rlp-derive/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
-## [Unreleased]
+## [0.2.0] - 2024-09-10
+- Updated `rlp` to 0.6. [#859](https://github.com/paritytech/parity-common/pull/859)
```

Comment:
this is a dev-dependency, so it shouldn't affect versioning, but there was another breaking unreleased change
```suggestion
```